addon/components/resize-sensor.js

```diff
@@ -0,0 +1,22 @@
+import Ember from 'ember';
+import layout from '../templates/components/resize-sensor';
+
+import ResizeSensor from 'resize-sensor';
+
+export default Ember.Component.extend({
+  layout,
+  classNames: ['resize-sensor-container'],
+  didInsertElement() {
+    const yieldHtml = this.$().children()[0];
+    this._resizeSensor = new ResizeSensor(yieldHtml, () => {
+      const onResize = this.get('onResize');
+      const $element = $(yieldHtml);
```

Comment:
Again, we want avoid using jquery. We should just send the element itself directly